src/components/gmServiceGmail.js

```diff
@@ -557,46 +557,49 @@ gmServiceGmail.prototype = {
           }
           
           try {
-            //var tbMatches = data.match(/\["tb",(?:.|\s)+?](?:\s]){2,}(?!\s,,)/g);   see: https://github.com/gmail-manager-community/gmail-manager-community/issues/4
-            var tbMatches = [];
+            var loc1 = data.indexOf("var VIEW_DATA=[[");
+            var loc2 = data.lastIndexOf("var GM_TIMING_END_CHUNK2");
+            var viewData = data.substring(loc1+14, loc2-2);
```

Comment:
@ath0mas, data.indexOf(string) returns the start position of string in data, so:
* after finding the start of ```"var VIEW_DATA=[["```, move move forward 14 characters to the start of the JSON (the first of the 2 ```[``` in that string)
* after finding ```"var GM_TIMING_END_CHUNK2"```, go back 2 characters to where the JSON ends

These offsets could have been in the ```var locx =``` lines, but it's the same end result.